Loops/task_3.py

```diff
@@ -0,0 +1,6 @@
+for i in range(1, 6):
+    print()
+    for x in range(1, 6):
+        print(x, end=' ')
+        if x == i:
+            break
```

Comment:
If I select range(1, 8) - program working incorrect(

`for i in range(1,6):
    for j in range(1, i+1):
        print(j, end=' ')
    print()`